### ISSUE TITLE
Problem importing lib v4.0.2

### ISSUE THREAD

**Issue by nelz9999** [launchdarkly/python-client#69]
Looks like maybe `python2.6-requirements.txt` isn't in the zip file on PYPI.

```
Searching for ldclient-py>=4.0.0
Reading https://pypi.python.org/simple/ldclient-py/
Downloading https://pypi.python.org/packages/4a/19/744c4fa7ab4116b3267de1ec132290b92fb4c24e39de0cb421aee6ae4fab/ldclient-py-4.0.2.tar.gz#md5=781fdd286c422f31155364c30c188ae0
Best match: ldclient-py 4.0.2
Processing ldclient-py-4.0.2.tar.gz
Writing /tmp/easy_install-RZ8Mp3/ldclient-py-4.0.2/setup.cfg
Running ldclient-py-4.0.2/setup.py -q bdist_egg --dist-dir /tmp/easy_install-RZ8Mp3/ldclient-py-4.0.2/egg-dist-tmp-bLgInv
Traceback (most recent call last):
  File "setup.py", line 86, in <module>
    include_package_data=True)
  File "/opt/circleci/python/2.7.12/lib/python2.7/distutils/core.py", line 151, in setup
    dist.run_commands()
  File "/opt/circleci/python/2.7.12/lib/python2.7/distutils/dist.py", line 953, in run_commands
    self.run_command(cmd)
  File "/opt/circleci/python/2.7.12/lib/python2.7/distutils/dist.py", line 972, in run_command
    cmd_obj.run()
  File "/home/ubuntu/observer/venv/lib/python2.7/site-packages/setuptools/command/install.py", line 67, in run
    self.do_egg_install()
  File "/home/ubuntu/observer/venv/lib/python2.7/site-packages/setuptools/command/install.py", line 117, in do_egg_install
    cmd.run()
  File "/home/ubuntu/observer/venv/lib/python2.7/site-packages/setuptools/command/easy_install.py", line 411, in run
    self.easy_install(spec, not self.no_deps)
  File "/home/ubuntu/observer/venv/lib/python2.7/site-packages/setuptools/command/easy_install.py", line 655, in easy_install
    return self.install_item(None, spec, tmpdir, deps, True)
  File "/home/ubuntu/observer/venv/lib/python2.7/site-packages/setuptools/command/easy_install.py", line 702, in install_item
    self.process_distribution(spec, dist, deps)
  File "/home/ubuntu/observer/venv/lib/python2.7/site-packages/setuptools/command/easy_install.py", line 747, in process_distribution
    [requirement], self.local_index, self.easy_install
  File "/home/ubuntu/observer/venv/lib/python2.7/site-packages/pkg_resources/__init__.py", line 851, in resolve
    dist = best[req.key] = env.best_match(req, ws, installer)
  File "/home/ubuntu/observer/venv/lib/python2.7/site-packages/pkg_resources/__init__.py", line 1123, in best_match
    return self.obtain(req, installer)
  File "/home/ubuntu/observer/venv/lib/python2.7/site-packages/pkg_resources/__init__.py", line 1135, in obtain
    return installer(requirement)
  File "/home/ubuntu/observer/venv/lib/python2.7/site-packages/setuptools/command/easy_install.py", line 674, in easy_install
    return self.install_item(spec, dist.location, tmpdir, deps)
  File "/home/ubuntu/observer/venv/lib/python2.7/site-packages/setuptools/command/easy_install.py", line 700, in install_item
    dists = self.install_eggs(spec, download, tmpdir)
  File "/home/ubuntu/observer/venv/lib/python2.7/site-packages/setuptools/command/easy_install.py", line 881, in install_eggs
    return self.build_and_install(setup_script, setup_base)
  File "/home/ubuntu/observer/venv/lib/python2.7/site-packages/setuptools/command/easy_install.py", line 1120, in build_and_install
    self.run_setup(setup_script, setup_base, args)
  File "/home/ubuntu/observer/venv/lib/python2.7/site-packages/setuptools/command/easy_install.py", line 1106, in run_setup
    run_setup(setup_script, args)
  File "/home/ubuntu/observer/venv/lib/python2.7/site-packages/setuptools/sandbox.py", line 258, in run_setup
    raise
  File "/opt/circleci/python/2.7.12/lib/python2.7/contextlib.py", line 35, in __exit__
    self.gen.throw(type, value, traceback)
  File "/home/ubuntu/observer/venv/lib/python2.7/site-packages/setuptools/sandbox.py", line 198, in setup_context
    yield
  File "/opt/circleci/python/2.7.12/lib/python2.7/contextlib.py", line 35, in __exit__
    self.gen.throw(type, value, traceback)
  File "/home/ubuntu/observer/venv/lib/python2.7/site-packages/setuptools/sandbox.py", line 169, in save_modules
    saved_exc.resume()
  File "/home/ubuntu/observer/venv/lib/python2.7/site-packages/setuptools/sandbox.py", line 144, in resume
    six.reraise(type, exc, self._tb)
  File "/home/ubuntu/observer/venv/lib/python2.7/site-packages/setuptools/sandbox.py", line 157, in save_modules
    yield saved
  File "/home/ubuntu/observer/venv/lib/python2.7/site-packages/setuptools/sandbox.py", line 198, in setup_context
    yield
  File "/home/ubuntu/observer/venv/lib/python2.7/site-packages/setuptools/sandbox.py", line 255, in run_setup
    DirectorySandbox(setup_dir).run(runner)
  File "/home/ubuntu/observer/venv/lib/python2.7/site-packages/setuptools/sandbox.py", line 285, in run
    return func()
  File "/home/ubuntu/observer/venv/lib/python2.7/site-packages/setuptools/sandbox.py", line 253, in runner
    _execfile(setup_script, ns)
  File "/home/ubuntu/observer/venv/lib/python2.7/site-packages/setuptools/sandbox.py", line 47, in _execfile
    exec(code, globals, locals)
  File "/tmp/easy_install-RZ8Mp3/ldclient-py-4.0.2/setup.py", line 22, in <module>
    cmd = ['git', 'describe']
  File "/home/ubuntu/observer/venv/lib/python2.7/site-packages/pip/req/req_file.py", line 84, in parse_requirements
    filename, comes_from=comes_from, session=session
  File "/home/ubuntu/observer/venv/lib/python2.7/site-packages/pip/download.py", line 425, in get_file_content
    'Could not open requirements file: %s' % str(exc)
pip.exceptions.InstallationError: Could not open requirements file: [Errno 2] No such file or directory: 'python2.6-requirements.txt'

```

**Comment by nelz9999** [comment]
https://pypi.python.org/packages/4a/19/744c4fa7ab4116b3267de1ec132290b92fb4c24e39de0cb421aee6ae4fab/ldclient-py-4.0.2.tar.gz#md5=781fdd286c422f31155364c30c188ae0

**Comment by drichelson** [comment]
Thanks @nelz9999. We're taking a look right now

**Comment by drichelson** [comment]
@nelz9999 we've fixed this in 4.0.3. Please contact us at support@launchdarkly.com if you have any more questions
Thanks
